airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/JdbcBufferedConsumerFactory.kt

```diff
@@ -63,16 +63,24 @@ object JdbcBufferedConsumerFactory {
         optimalBatchSizeBytes: Long = DEFAULT_OPTIMAL_BATCH_SIZE_FOR_FLUSH,
         parsedCatalog: ParsedCatalog,
     ): SerializedAirbyteMessageConsumer {
-        val writeConfigs =
-            createWriteConfigs(
-                namingResolver,
-                config,
-                sqlOperations.isSchemaRequired,
-                parsedCatalog
+        if (sqlOperations.isSchemaRequired) {
+            Preconditions.checkState(
+                config.has("schema"),
+                "jdbc destinations must specify a schema."
             )
+        }
+        val writeConfigs = mutableListOf<WriteConfig>()
```

Comment:
```suggestion
        // This list is populated when the OnStartFunction is invoked
        // which is OK, because it's only used by the flush function
        // which never runs until after OnStart runs
        val writeConfigs = mutableListOf<WriteConfig>()
```

( :cry: )